ssas/systems.go

```diff
@@ -56,6 +56,7 @@ type System struct {
 	EncryptionKeys []EncryptionKey `json:"encryption_keys,omitempty"`
 	Secrets        []Secret        `json:"secrets,omitempty"`
 	LastTokenAt    time.Time       `json:"last_token_at"`
+	XData          string          `json:"xdata"`
```

Comment:
Should the json name be `x_data` or `xData` instead?

Reply:
I tried to keep it the same as the group resource.